src/game/behaviors/sparkle_spawn_star.inc.c

```diff
@@ -16,7 +16,7 @@ void bhv_spawned_star_init(void) {
     if (!(o->oInteractionSubtype & INT_SUBTYPE_NO_EXIT)) {
         o->oBehParams = o->parentObj->oBehParams;
     }
-    s32 param = GET_BPARAM1(o->oBehParams);
+    u32 param = GET_BPARAM1(o->oBehParams);
```

Comment:
not too related but i feel like this should be a u8 for consistency with the other star code, and why is there no GLOBAL_STAR_IDS handling for this?

Reply:
true, nice catch